LocalyticsReactTest/android/app/build.gradle

```diff
@@ -134,11 +145,11 @@ android {
 
 dependencies {
     compile project(':localytics-react-native')
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile 'com.android.support:appcompat-v7:23.0.1'
-    compile 'com.facebook.react:react-native:+'  // From node_modules
     compile 'com.google.android.gms:play-services-gcm:9.4.0'
     compile 'com.google.android.gms:play-services-location:9.4.0'
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.facebook.react:react-native:+"  // From node_modules
```

Comment:
if you're gonna change some to implementation, why not all?

Reply:
Didn't manually change any of this. This is what the react scripts did when I followed their upgrade instructions.